tools/static-assets/skel-ts-apollo/server/apollo.ts

```diff
@@ -0,0 +1,43 @@
+import { ApolloServer, ExpressContext } from 'apollo-server';
```

Comment:
This doesn't look correct. It should be imported from `@apollo/server`. You are mixing here Apollo server v4 and v3.